plugins/com.google.cloud.tools.eclipse.dataflow.ui/helpContexts.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE contexts [
+
+  <!ENTITY group-id "&lt;b&gt;Group ID:&lt;/b&gt; The Maven Group ID. Should be an alphanumeric path separated by periods. Typically the same as the Java package name.">
+
+  <!ENTITY artifact-id "&lt;b&gt;Artifact ID:&lt;/b&gt; The Maven Artifact ID. Should be an alphanumeric name separated by dashes.">
+
+]>
+<contexts>
+  <context id="NewDataflowProjectContext" title="Create Google Cloud Dataflow Project">
+    <description>This wizard creates a new Eclipse project for Google Cloud Dataflow.
+
+      &group-id;
+
+      &artifact-id;
+      
+      &lt;b&gt;Project template:&lt;/b&gt; @tgroh please suggest text
+      
+      &lt;b&gt;Dataflow version:&lt;/b&gt; 1.9.x is ????. 2.x.x is based on Apache Beam.
```

Comment:
I think this might be better to be just a strong recommendation to use the latest 2.x version. 1.9.x mainly exists for users with compatibility reasons to select it, but they will likely already know what version they want to use.

Reply:
Suggested text?

Reply:
Ping